tools/README.md

```diff
@@ -334,6 +334,35 @@ Shuffle Skew Check: (When task's Shuffle Read Size > 3 * Avg Stage-level size)
 |1       |2      |0             |2224  |1      |222.22         |8.8           |3333.33          |111.11          |0.01            |false     |ExceptionFailure(ai.rapids.cudf.CudfException,cuDF failure at: /dddd/xxxxxxx/ccccc/bbbbbbbbb/aaaaaaa|
 +--------+-------+--------------+------+-------+---------------+--------------+-----------------+----------------+----------------+----------+----------------------------------------------------------------------------------------------------+
 ```
+#### C. Health Check
+
+```
+Application application_1616746343401_0025 (index=1) failed tasks:
++-------+--------------+------+-------+------------------------------------+
+|stageId|stageAttemptId|taskId|attempt|endReason_first36char               |
```

Comment:
any reason to use 36 char here and 100 below? seems like we could use 100 in both.  Maybe put ... at the end of hte string if its longer.